.github/ISSUE_TEMPLATE/new-feature-request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: New Feature Request
+about: Suggest a new feature for this project
```

Comment:
Tone is a little flat given the rest of the descriptions. How about "Have an idea that could improve this project? We want to hear it!" ?